templates/install/macos.md

```diff
@@ -30,13 +30,27 @@ Given that GitHub Actions does [not yet support builds on Apple
 ARM](https://github.com/actions/virtual-environments/issues/2187), installation
 of Lean is for the moment a bit more complex.
 
-Specifically, `elan` – which is otherwise recommended (and installed)
+Specifically, `elan` – which is installed
 as part of the above instructions – will not be able to fetch Lean binaries on
-these devices.
+these devices if installed the normal way.
 
-Until [a separate M1 installation is
-automated](https://github.com/leanprover-community/mathlib-tools/issues/107),
-manual compilation of Lean is required on Apple ARM hardware.
+The following instructions are adapted from [Fedor Pavutnitskiy](https://leanprover.zulipchat.com/#narrow/stream/113489-new-members/topic/M1.20Macs.3A.20Installing.20the.20Lean.203.20toolchain/near/262832039), and allow you to install elan through [Rosetta](https://developer.apple.com/documentation/apple-silicon/about-the-rosetta-translation-environment).
+
+1. Open a new terminal window and install XCode Command Line Tools and Rosetta 2 using `xcode-select --install` and `softwareupdate --install-rosetta`.
+
+2. We will install a second, separate x86 installation of Homebrew, which is easiest done by running a shell entirely using Rosetta 2. Do so by running `arch -x86_64 zsh`. The remainder of the commands below should be run from within this `x86`-running window, though once the steps have been completed, the installed tools will work in any future shell.
+
+3. Install a second installation of Homebrew for `x86` with `/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"`. It will automatically install itself into a second location (`/usr/local`, rather than `/opt/`).
+
+4. Follow the same steps described in [Controlled Installation for macOS](https://leanprover-community.github.io/install/macos_details.html) using the `brew` you just installed:
+
+```
+/usr/local/bin/brew install elan mathlibtools
+elan toolchain install stable 
+elan default stable  
+```
+
+5. Install Visual Studio Code and the Lean extension via `brew install --cask visual-studio-code && code --install-extension jroesch.lean`.
```

Comment:
Does it matter which `brew` you use here if you have both architectures installed?
```suggestion
5. Install Visual Studio Code and the Lean extension via `/usr/local/brew install --cask visual-studio-code && code --install-extension jroesch.lean`.
```
or:
```suggestion
5. Install Visual Studio Code and the Lean extension via `brew install --cask visual-studio-code && code --install-extension jroesch.lean` (both the x86 and ARM versions of `brew` will work).
```

Reply:
I imagine both should work - downloading from the website works too - but I haven't tested it.